src/editor/codeeditor.cpp

```diff
@@ -42,26 +46,28 @@ CodeEditor::CodeEditor(QWidget *parent, const QString &fileName) : QPlainTextEdi
 
     //create objects connected to codeEditor
     mLineNumberArea = new LineNumberArea(this);
-    mTimer = new QTimer;
-    mLcpp = new LexerCPP();
-    mTimer = new QTimer;
+    //mTimer = new QTimer;
```

Comment:
what is the purpose of commenting some line here?

Reply:
I had created a version where codeEditor.cpp was withot commented lines of code. But after that I realized that some of my functional doens't work. I should have gone to the previous commit and as you can see in that commit CodeEditor.cpp wasn't formated. 
I'll fix it